### DISCUSSION TITLE
How to remove the blue box?

### DISCUSSION THREAD

**Discussion by danmartinvela** [zed-industries/zed#13553] [Feature Requests]
This blue box looks ugly, any way to remove it by settings?
![image](https://github.com/zed-industries/zed/assets/74007579/96a3e7a9-220b-4a33-975c-52fce0177148)

**Comment by tepavcevic** [comment]
IIRC from some other discussion or issue, that outline is accessibility feature and as of right now is not possible to remove it.

**Comment by dimitrisnl** [comment]
Take a look here https://github.com/zed-industries/zed/discussions/12725

In short they are using a `text` related style for this a11y border, so it looks out of place.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
